app/services/create_planned_disbursement.rb

```diff
@@ -11,6 +11,8 @@ def call(attributes: {})
 
     planned_disbursement.parent_activity = activity
     planned_disbursement.assign_attributes(attributes)
+    planned_disbursement.planned_disbursement_type = "1"
```

Comment:
FYI you can pass a symbol i.e. `:original` or `:revised` when setting this which is a little more readable.